### PR TITLE
idl 4.2 [4991]

### PR DESCRIPTION
**Pull request by LuisGP**

### CODE REVIEW

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/49/)

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/50/)

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/52/)

File: docs/dynamictypes.rst [hunk 1]
```diff
@@ -207,33 +237,42 @@ Structure
...
 will change its id to the next value.
-To get the id of a member by name, ``DynamicData`` exposes the method ``GetMemberIdByName``.
+To get the id of a member by name, ``DynamicData`` exposes the method ``get_member_id_by_name``.
 
 .. literalinclude:: ../code/CodeTester.cpp
    :language: c++
    :start-after: //DYNAMIC_TYPES_CREATE_STRUCTS
    :end-before: //!--
 
+Structures allows inheritance, exactly with the same OOP meaning. To inherit from another structure, we must create the
```
Comment (richiware) [inline docs/dynamictypes.rst]:
```suggestion
Structures allow inheritance, exactly with the same OOP meaning. To inherit from another structure, we must create the
```

File: docs/dynamictypes.rst [hunk 2]
```diff
@@ -175,26 +174,57 @@ with ``UINT32`` or with strings using the names of the members added to the buil
 Bitset
 ^^^^^^
 
-Bitset types emulate a list of boolean values but optimized for space allocation
-using each bit for a different value.
-They work like a ``boolean`` type with the only difference that the ``GetBoolValue`` and
-``SetBoolValue`` need the index of the bit that users want to read or write.
+Bitset types are similar to `structure` types but its members are only `bitfields`, which are stored optimally.
+In the static version of bitsets, each bit uses just one bit in memory (with platform limitations) without alignment
```
[the remark this answers is not in the document]
Comment (LuisGP) [reply docs/dynamictypes.rst]: It's inspired by a phrase in the IDL 4.2 document: "Stored optimally" means that one bit uses just one bit in memory. "Concatenated" means
that each bit field will be placed in memory just after its predecessor within the bit set (no alignment considerations
apply).

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/56/)

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/70/)

**Comment by LuisGP** [comment]
Closed because integrated into #31

**Pull request closed without merging**